docs/operate/validators/run-validators.mdx

```diff
@@ -246,7 +246,7 @@ docker run \
   ./validator \
   --db /hyperlane_db \
   --originChainName <your_chain_name> \
-  --reorgPeriod 20 \
+  --reorgPeriod 1 \
```

Comment:
should update the non-docker example too, line 279

Reply:
yes pls